### ISSUE TITLE
Descriptions not working - Intermittent problem

### ISSUE THREAD

**Issue by para2022** [WWBN/AVideo#8309]
3 days ago after a git pull my descriptions have stopped working. Some users have descriptions others do not even though when i go to edit their movie the description is there in the edit page. I save it again and still it will not show. Obviously i have some users moaning to me about it. Ive cleared cache restarted all services and nothing helps. No errors in logs. Descriptions were working fine prior to the update.

As a side note have you removed the username from the videos on the homepage? I got users moaning about that now also as users do not know whos uploaded the video's on the home page. I notice you need to click 3 dots for the username to show on a dropdown of each video. This is a poor move if it has been changed. Much better how it was and i dont understand the rationale for removing the uploaders name from homepage.

**Comment by para2022** [comment]
Ok i can see on the demo site names were removed but i seriously think that is a bad move and is sure to piss a lot of people off. I have never seen on any platform where you cant see who uploaded a video. Makes no sense and its another way to drop channel views.

**Comment by DanielnetoDotCom** [comment]
I agree the channel name should be always visible, soon you I will send and update with the channel name visible

**Comment by para2022** [comment]
Thanks mate and please dont think im having a pop at you cos you do such amazing work with the platform. But that change did not make any sense at all which is why i mentioned it as i couldn't get my head around it. cheers.

**Comment by para2022** [comment]
Here is a vid with missing description https://ourtube.co.uk/v/49633?channelName=Nature

Here is the edit content:

![Screenshot 2023-08-13 at 00 35 32](https://github.com/WWBN/AVideo/assets/107706817/41bd8eb5-d7ba-4fd5-9acc-1ab111fa7346)

**Comment by para2022** [comment]
User comments:

my notifications are messing up, plus few other little glitches.
don`t like new layout with no show names

--------------------

My descripts are missing which is half this chans popularity🙂 half the time hehe.(psst I`m glad you can see it too tho)

**Comment by DanielnetoDotCom** [comment]
the descriptions I do not know what is happening. maybe the HTML in it. 

can you please try a simple description and see if it works?

or paste your description here so I can try on my side.

**Comment by para2022** [comment]
Its not the html cos i cut his description and replaced it with 

testing 1 2 3 and it did not show either. He nor me can get a description to show no matter what is inputted there.

**Comment by DanielnetoDotCom** [comment]
I have no idea, can you please try to reproduce it on the demo site?

**Comment by para2022** [comment]
Here is another https://ourtube.co.uk/video/49785/florida-heat-wave-more-important-shit-you-need-to-know?channelName=BillyJones

He has no html. there are loads of them now and they are random. Some have working descriptions and others dont. Here is his description

#news #politics #secret #trump #movie #bioweapon #military #japan #janglesteined #top #virus #nipah #marburg #TRUMP #RUSSIA #MOVIE #PUTIN #UKRAINE #JESUS #REVELATION #PEE #WEE #HERMAN #COPPERPOT #JANGLESTEINED #MKULTRA #CHEMTRAILS #WEATHER #WARFARE #VIRUS #MARBURG #NIPAH #EVENT201 #Q #QANON #SONG #ALAN #PARSONS #SATAN #BAPHOMET #AI #KILLGRID #SMART #CITIES #CITY #JESUS #BUDDHA #ALAN #WATTS #BILL #COOPER #BILLY #JONES #RESIST 3JANGLESTEINED #FIRE #BRAIN #HEAD #CIA #DRUGS #BRAINWASHING #LIES #MIND #CONTROL #JOSE #DELGADO #leaked #mcafee #haarp #aurora #are51 #ufo #aliens #PHIL #SCHNEIDER #BOB #LAZAR #BILL #COOPER #JANGLESTEINED #RESIST #BILLY #JONES #STEALTH #CLOAKING #INVISIBLE #CGI #BLUEBEAM #COPPERPOT #BAPHOMET #MIKE #TYSON #DANCE #VAXXED #CHIPPED #TOMMY #ROBINSON #ALEX #JONES #AMANDA #VOLLMER #TENPENNY #BMX #CYCLING #AI #BIKE #BIKING #CYCLING #TREK #BOYCOTT

reposted.mirrored
for education only

**Comment by DanielnetoDotCom** [comment]
I found the issues

**Comment by para2022** [comment]
Thats fab! the descriptions are showing now. Also the usernames and avatars looks great, nice touch. theres a slight alignment issue where the upload time goes, for instance '1 hour ago' It may show more clearer on my front page. https://ourtube.co.uk

**Comment by para2022** [comment]
maybe move the time next to the dots on the right so it aligns nicely. Also you could make the round image a slightly smaller rounded square like on the video view page (a smaller version of that) it may also align a bit better and offers more consistency. Just some ideas :)

**Comment by para2022** [comment]
btw that last update made the layout look soo much better. nice one dan.
